code/modules/events/ion_storm.dm

```diff
@@ -2,9 +2,14 @@
 	name = "Ion Storm"
 	typepath = /datum/round_event/ion_storm
 	weight = 15
-	min_players = 2
+	min_players = 0 // Handled by canSpawnEvent override
 	can_malf_fake_alert = TRUE
 
+/datum/round_event_control/ion_storm/canSpawnEvent(players_amt, gamemode)
+	. = ..()
+	if(players_amt < CONFIG_GET(number/malf_ai_minimum_pop))
+		return FALSE
```

Comment:
Dont remove this event entirely, just make it so only the malf event doesn't occur (Also I think this is the wrong event, since this doesn't seem to be the one that gives AIs traitor)

Reply:
The variant of the ion storm event that makes the AI into a traitor is a dynamic ruleset called `/datum/dynamic_ruleset/midround/malf`, not the actual ion storm event

Reply:
Fixed.